src/angular/src/app/pages/settings/settings-page.component.html

```
@@ -68,6 +68,61 @@ <h3 class="card-header collapsible-header" (click)="toggleAdvancedLftp()">
+              <app-option
+                [type]="option.type"
+                [label]="option.label"
+                [description]="option.description"
+                [choices]="option.choices || []"
+                [value]="getOptionValue(config$ | async, option.valuePath)"
+                (changeEvent)="onSetConfig(option.valuePath[0], option.valuePath[1], $event)">
+              </app-option>
+            </div>
+          }
+          <div class="test-connection">
+            <button class="btn btn-sm btn-outline-secondary" [disabled]="sonarrTesting" (click)="onTestSonarr()">
```

HTMLHint (1.9.2)

[warning] The type attribute must be present on <button> elements.

(button-type-require)